pennylane/qcut/tapes.py

```diff
@@ -102,7 +102,7 @@ def tape_to_graph(tape: QuantumTape) -> MultiDiGraph:
 
 
 # pylint: disable=protected-access
-def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
+def graph_to_tape(graph: MultiDiGraph) -> QuantumScript:
```

Comment:
# Thought
It looks like this function signature was previously incorrect. Hopefully, the change won't break any existing code.